Header header
bool ball_detected
float64 ball_range
float64 ball_bearing
float64 ball_x
float64 ball_y

#float64 ball_center_x
#float64 ball_center_y
#float64 ball_z
